releng/com.b2international.snowowl.server.update/assembly/common/scripts/snowowl_export.sh

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env bash
+#
+# Copyright 2018 B2i Healthcare Pte Ltd, http://b2i.sg
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#
+# Snow Owl terminology server deploy script
+# See usage or execute the script with the -h flag to get further information.
+#
+
+# The user for the user authentication for the REST API.
+SNOW_OWL_USER=""
+
+# The password for the above user.
+SNOW_OWL_PASSWORD=""
+
+# The export type to use for the export config.
+EXPORT_TYPE="DELTA"
+
+# Forces the export config to export concepts and relationships only.
+IS_CONCEPT_AND_RELATIONSHIPS_ONLY=${false}
+
+# The target folder to save the exported snapshot.
+TARGET_FOLDER=""
+
+# The server where the export should be initiated on
+TARGET_ENVIRONMENT=""
+
+# UUID of the snapshot export.
+EXPORT_UUID=""
+
+# Media type to use for the export configuration.
+MEDIA_TYPE="application/vnd.com.b2international.snowowl+json"
+
+# Accept header for archive download.
+ACCEPT_HEADER="application/octet-stream"
+
+# The address where the export config endpoint can be found.
+EXPORT_CONFIG_POST_ENDPOINT="snowowl/snomed-ct/v3/exports"
```

Comment:
Snow Owl actual API endpoint should be configurable via a `SNOWOWL_BASE_URL` variable (to support earlier versions of the export API).